docker-compose.yml

```diff
@@ -26,7 +26,7 @@ services:
       - postgres_data:/var/lib/postgresql/data
 
   signal:
-    image: bbernhard/signal-cli-rest-api:0.67
+    image: bbernhard/signal-cli-rest-api:0.119-dev
```

Comment:
@mattwr18 Are we going to roll our own image or not?

Reply:
Don't think so. There is already a pre-release https://github.com/bbernhard/signal-cli-rest-api/releases/tag/0.68-pre

There, he advises to use this image for testing things out.